src/pages/records.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import NavigationBar from '../components/header';
+import TitleImage from '../components/title-image';
+import RecordsImage from '../images/records.jpg';
+import theme from '../config/theme';
+import PageBody from '../components/page-body';
+import Table from '../components/table';
+import Card from '../components/card';
+import WiderContainer from '../components/wider-container';
+
+
+export default class RecordsPage extends React.Component {
+
+  render() {
+    return (
+      <>
+        <NavigationBar />
```

Comment:
Can you switch this to use layout? (like [here](https://github.com/Brown-Running-Club/website/blob/2ac7f5f750eafdbc22b36c103f304551ab7eb94b/src/pages/about.tsx#L13))
The page title isn't updating